Translations/ru.lproj/InfoPlist.strings

```diff
@@ -9,7 +9,7 @@
 "XZ Archive" = "Архив XZ";
 "XZ Tarball Archive" = "Сжатый TAR-архив XZ";
 "RAR Archive" = "Архив RAR";
-"RAR Legacy Archive Volume" = "RAR Legacy Archive Volume";
+"RAR Legacy Archive Volume" = "Архив RAR старого формата";
 "Generic Archive Volume" = "Generic Archive Volume";
```

Comment:
Can you also translate this one also? This refers to any parted archive, like for example file.7z.001 ... file.7z.099.